setup.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+from setuptools import setup, find_packages
+
+setup(name='Protego',
+      version='0.1dev',
+      description='Pure-Python robots.txt parser with support for modern conventions',
+      long_description=open("README.md").read(),
+      long_description_content_type='text/markdown',
+      author='Anubhav Patel',
+      author_email='anubhavp28@gmail.com',
+      license='BSD',
+      package_dir={'': 'src'},
+      packages=find_packages('src'),
+      py_modules=['protego'],
+      python_requires='>=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*',
+      install_requires=['six'],
+      tests_require=['pytest'],
+      include_package_data=True,
+      keywords=['robots.txt', 'parser', 'robots', 'rep'],
+      classifiers=[
+          'Framework :: Scrapy',
```

Comment:
I think you can remove this line, in the end Protego is quite independent of Scrapy.